src/App.tsx

```diff
@@ -22,10 +22,10 @@ import "./App.css";
 window.addEventListener("unload", function () { });
 
 type AppProps = {
-	IEOREDGE: boolean,
-	currYear: number,
-	firebase?: any,
-	ContentEditor: React.FC<ContentEditorProps>
+    IEOREDGE: boolean,
```

Comment:
Can you set it so the auto format uses a tab instead of four spaces? It would also get rid of these unnecessary changes in the commit.

Reply:
Changed vscode settings to use indentations instead of spaces, then saved the file.